models/aws_project.rb

```diff
@@ -54,7 +54,7 @@ def daily_report(date=(Date.today - 2), slack=true, text=true, rerun=false, verb
       return
     end
 
-    record_instance_logs(rerun) if date >= Date.today - 2 && date <= Date.today
+    record_instance_logs(rerun) if date >= DEFAULT_DATE && date <= Date.today
```

Comment:
I might have missed something; what is this conditional for?

Reply:
As we can only record instance logs for today, this is checking that we only create/ recreate them if we are generating the 'latest' report (I can't recall the reason this is a range so have removed this). E.g. if I decide to run a report for a date last week there's no reason for it to generate/rewrite today's instance logs.

Reply:
I was missing something; I forgot that instance logs aren't bound by temporal instability (:mage:).